src/game/engine/radio.cpp

```diff
@@ -13,17 +13,18 @@
  *            LICENSE
  */
 #include "radio.h"
+#include "techno.h"
 #include "target.h"
 
-// A invalid target instance parsed into targets as a default param.
-static target_t LParam = 0;
-
+ // A invalid target instance parsed into targets as a default param.
 target_t &RadioClass::Get_LParam()
 {
-    return LParam;
+    static target_t _lparam = 0;
+    return _lparam;
 }
 
-const char *RadioClass::Messages[RADIO_COUNT] = {
+#ifdef CHRONOSHIFT_DEBUG
```

Comment:
I don't think there is any need for the array to be behind the debug macro, if its compiled in but never used, what does it matter?